ironfish/src/node.ts

```diff
@@ -318,6 +318,8 @@ export class IronfishNode {
     // so we should start it as soon as possible
     this.workerPool.start()
 
+    await this.memPool.feeEstimator.init(this.chain)
```

Comment:
This looks a bit different than the others now. We used to have

- open: open any DB connections and load initial data (should not depend on other systems)
- close: close any db connections and unload any data
- start: start any active event loops and systems that passively run
- start: stop any active event loops and systems that passively run

Reply:
Ah it sounds like this would be a better fit in `open`. I'll see what it takes to move it there